src/storage.cc

```diff
@@ -75,9 +75,11 @@ void Storage::EnterNotify() {
   buy_policy.Start();
 
   if (out_commods.size() == 1) {
-    sell_policy.Init(this, &stocks, std::string("stocks"))
-        .Set(out_commods.front())
-        .Start();
+    sell_policy.Init(this, &stocks, std::string("stocks"), 1e+299, false, sell_quantity)
+      .Set(out_commods.front())
+      .Start();
+    }
+    
   } else {
```

Comment:
test is one to many `{` in there

Reply:
```suggestion

  } else {
```

Reply:
this is correct but it is your PR :)
so you could make the change directly :)

Reply:
Thanks! I made the suggestion and added the commit to batch on the website, but apparently never actually committed it...